### PR TITLE
fix bug: Table content is misplaced if there is empty column in it.

### PR DESCRIPTION
**Pull request by vwxyzh**
@chenkennt @vicancy @hellosnow @ansyral @qinezh @superyyrrzz

### CODE REVIEW

**Comment by hellosnow** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
:clock1:

File: src/Microsoft.DocAsCode.MarkdownLite/Tables/MarkdownTableBlockRule.cs [hunk 1]
```diff
@@ -28,13 +28,11 @@ public virtual IMarkdownToken TryMatch(IMarkdownParser parser, IMarkdownParsingC
             }
             var sourceInfo = context.Consume(match.Length);
 
-            var rows = match.Groups[3].Value.ReplaceRegex(Regexes.Lexers.UselessGfmTableCell, string.Empty).Split('\n').Select(x => x).ToArray();
+            var rows = match.Groups[3].Value.ReplaceRegex(Regexes.Lexers.UselessGfmTableCell, string.Empty).Split('\n');
             var cells = new string[rows.Length][];
             for (int i = 0; i < rows.Length; i++)
             {
-                var columns = rows[i]
-                  .ReplaceRegex(Regexes.Lexers.EmptyGfmTableCell, string.Empty)
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.MarkdownLite/Tables/MarkdownTableBlockRule.cs]: >Regexes.Lexers.EmptyGfmTableCell [](start = 32, length = 32)

Can be removed from Regexes?

**Comment by qinezh** [comment]
:shipit:

**Pull request merged**